angular.json

```diff
@@ -26,14 +26,17 @@
             "styles": [
               "styleguide/styles.scss",
               "node_modules/leaflet/dist/leaflet.css",
-              "node_modules/leaflet-draw/dist/leaflet.draw.css"
+              "node_modules/@geoman-io/leaflet-geoman-free/dist/leaflet-geoman.css"
```

Comment:
I would much rather see an additional angular library package that bridges the gap between the use of this library. (for example ngx-leaflet-draw) Doing this won't pollute the angular.json of the ACPAAS ui and separates the versioning & adapting of the library.